apps/financial-aid/web-osk/src/components/Timeline/timeline.tsx

```diff
@@ -1,64 +1,78 @@
 import format from 'date-fns/format'
+import * as styles from './timeline.css'
+import cn from 'classnames'
 
 interface Props {
   state: ApplicationState
+  modified: string
   created: string
 }
 
-const Timeline = ({ state, created }: Props) => {
+const Timeline = ({ state, modified, created }: Props) => {
   const sections = [
     {
       name: 'Umsókn móttekin',
-      children: [
-        {
-          type: 'SUB_SECTION',
-          name: format(new Date(created), 'dd/MM/yyyy hh:mm'),
-        },
-      ],
+      text: 'Umsóknin verður tekin til úrvinnslu eins fljótt og kostur er.',
+      state: [ApplicationState.NEW],
+      date: format(new Date(created), 'dd/MM/yyyy HH:MM'),
     },
     {
       name: 'Umsókn í vinnslu',
-      children: [
-        {
-          type: 'SUB_SECTION',
-          name:
-            'Vinnsluaðili verður í sambandi ef þörf er á frekari upplýsingum',
-        },
-      ],
+      text:
+        'Úrvinnsla umsóknarinnar er hafin. Ef þörf er á frekari upplýsingum eða gögnum mun vinnsluaðili óska eftir því hér á þessari stöðusíðu.',
+      state: [ApplicationState.INPROGRESS, ApplicationState.DATANEEDED],
+      date: format(new Date(modified), 'dd/MM/yyyy HH:MM'),
     },
     {
       name: 'Niðurstaða',
-      children: [
-        {
-          type: 'SUB_SECTION',
-          name:
-            'Umsókn verður samþykkt eða synjuð og umsækjandi látinn vita um niðurstöðuna',
-        },
-      ],
+      text:
+        'Umsókn verður samþykkt eða synjuð og umsækjandi látinn vita um niðurstöðuna',
+      state: [ApplicationState.REJECTED, ApplicationState.APPROVED],
+      date: format(new Date(modified), 'dd/MM/yyyy HH:MM'),
     },
   ]
 
+  const [activeState, setActiveState] = useState(
+    sections.findIndex((el) => el.state.includes(state)),
+  )
+
   return (
     <Box marginY={[4, 4, 5]}>
       <Text as="h3" variant="h3" marginBottom={[1, 1, 2]}>
         Umsóknarferlið
       </Text>
-      <Text marginBottom={1}>
+      <Text marginBottom={[2, 2, 4]}>
         Hér geturðu séð hvað hefur gerst og hvað er framundan. Hikaðu ekki við
         að senda okkur athugasemd ef þú telur eitthvað óljóst eða rangt.
       </Text>
 
-      <FormStepper
-        activeSection={getActiveSectionForTimeline[state]}
-        sections={sections}
-      />
+      {sections.map((item, index) => {
+        return (
+          <Box
+            className={cn({
+              [`${styles.timelineContainer}`]: true,
+              [`${styles.activeState}`]: activeState >= index,
+              [`${styles.lineDown}`]: index !== sections.length - 1,
+            })}
+          >
+            <Box paddingLeft={3}>
+              <Text variant="h5">{item.name}</Text>
+              <Text marginBottom={2}>{item.text}</Text>
+
+              <Text variant="small" color="dark300" marginBottom={5}>
+                {(index === 0 || index === activeState) && item.date}
+              </Text>
```

Comment:
Do you want to do the boolean check on line 69 instead so we don't have the bottom margin if this is null?

Reply:
No the margin is nice here is example, so the spacing is always the same 
without margin:
![Screenshot 2021-11-19 at 15 46 50](https://user-images.githubusercontent.com/62568905/142651118-5bc672b7-7d9b-47b3-8cb5-0a20a06236c9.png)

with margin: 
 
![Screenshot 2021-11-19 at 15 47 07](https://user-images.githubusercontent.com/62568905/142651167-bbe98ff9-711d-4517-8f26-00a5812fbfd3.png)

I think the same space is nicer, what do you think? 👨‍🎨

Reply:
Agree, just looked strange in code.